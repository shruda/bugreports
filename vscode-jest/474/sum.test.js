
// Copyright 2004-present Facebook. All Rights Reserved.

const sum = require('./sum');

describe.each([[1, 2, 3], [1, 4, 5], [2, 6, 8]])(
    '.sum(%i, %i)',
    (a, b, expected) => {
      test(`returns ${expected}`, () => {
        expect(sum(a, b)).toMatchSnapshot(expected);
      });
  
      test(`returned value not be greater than ${expected}`, () => {
        expect(a + b).not.toBeGreaterThan(expected);
      });
  
      test(`returned value not be less than ${expected}`, () => {
        expect(a + b).not.toBeLessThan(expected);
      });
    },
  );