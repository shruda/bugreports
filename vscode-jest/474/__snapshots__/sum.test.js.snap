// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`.sum(1, 2) returns 3 1`] = `3`;

exports[`.sum(1, 4) returns 5 1`] = `5`;

exports[`.sum(2, 6) returns 8 1`] = `8`;
